packages/rx_bloc_cli/mason_templates/bricks/lib_social_logins/__brick__/lib/lib_social_logins/blocs/social_login_bloc.dart

```diff
@@ -47,7 +53,13 @@ class SocialLoginBloc extends $SocialLoginBloc {
   ConnectableStream<bool> _mapToLoggedInState() => _$loginEvent
       .throttleTime(const Duration(seconds: 1))
       .switchMap(
-        (_) => _socialLoginService.login().then((_) => true).asResultStream(),
+        (_) =>
+            _socialLoginService.login().then((_) => true).catchError((error) {
+          print(error);
+          if (error is CancelledErrorModel) {
+            return false;
```

Comment:
Please use syntax sugar

catchError((error) => error is! CancelledErrorModel))